src/ray/common/scheduling/cluster_resource_scheduler.cc

```diff
@@ -160,6 +196,7 @@ bool ClusterResourceScheduler::SubtractNodeAvailableResources(
   }
   NodeResources &resources = it->second;
 
+  // RAY_LOG(ERROR) << "   SUBTRACT RESOURCES: " << TaskRequestPrint(task_req);
```

Comment:
Please remove stray log statements, here and elsewhere.

Reply:
done